codecov/__init__.py

```diff
@@ -982,8 +985,11 @@ def main(*argv, **kwargs):
                 if codecov.gcov_args:
                     cmd.append(sanitize_arg("", codecov.gcov_args or ""))
                 cmd.append(path)
-                write("    Executing gcov (%s)" % cmd)
-                write(try_to_run(cmd))
+                if codecov.gcov_out:
+                    write("    Executing gcov (%s)" % cmd)
+                gcov_out = try_to_run(cmd)
+                if codecov.gcov_out:
```

Comment:
Shouldn't this be `if not codecov.gcov_out`?

Reply:
Because of the reverse nature of the value, it would make more sense if `gcov_out` was renamed.

Reply:
Fixed.